### ISSUE TITLE
Vector Tile Layer "Light Unit VT" seems to be broken

### ISSUE THREAD

**Issue by JakobMiksch** [compassinformatics/cpsi-mapview#186]
This `VT` Layer is not working. In the viewer it does not stop loading.

```
    {
      "layerType": "vt",
      "text": "Light Unit VT",
      "layerKey": "LIGHT_UNIT_VT",
      "url": "https://plmonaghandev.compass.ie/mapserver/?mode=tile&tilemode=gmap&tile={x}+{y}+{z}&layers=LightUnit&map.imagetype=mvt",
      "openLayers": {
        "visibility": false
      },
      "format": "MVT",
      "stylesBaseUrl": "https://plmonaghandev.compass.ie/resources/data/styling/",
      "styles": [ "LightUnit_Type.xml", "LightUnit_Owner.xml" ],
      "stylesForceNumericFilterVals": true
    },
```

Console output:

![image](https://user-images.githubusercontent.com/15704517/75480087-f0ee2600-59a0-11ea-8180-b9a9a98e7975.png)

However, the `VT WMS` Layer, which comes from the same source, does work.

I need the `VT`layer for testing #159

**Comment by geographika** [comment]
@JakobMiksch - I think we can probably remove this layer from the JSON file. To get vector tiles working from MapServer @chrismayer implemented an approach using the WMS protocol - see #120. See "layerKey": "LIGHT_UNIT_MVT" layer.

**Comment by chrismayer** [comment]
Closed via #187
